### ISSUE TITLE
Regression in 8.1.0 due to ACEP-patch (refresh fails on e-paper)

### ISSUE THREAD

**Issue by bablokb** [adafruit/circuitpython#7734]
### CircuitPython version

```python
Adafruit CircuitPython 8.1.0-beta.0 on 2023-03-01; Pimoroni Badger 2040 with rp2040
```

### Code/REPL

```python
na
```

### Behavior

see below

### Description

PR https://github.com/adafruit/circuitpython/commit/931c7c1c51fe3b2affa765d94c7178ae3d1be80c (ACEP) changed the interface to the EPaperDisplay class replacing the (single byte) refresh_display_command with a refresh_sequence. Since this PR the Badger2040 flashes during refreshes but the screen remains black.

**Comment by bablokb** [comment]
Fixed this by setting grayscale to True in the constructor. But this seems to be more a workaround than a fix, because the display is monochrome and not a grayscale display. So the switch to a refresh-sequence does  not seem to be the root cause after all.

**Comment by tannewt** [comment]
The code considers monochrome to be grayscale (versus having color.) What driver code are you using? It should set grayscale correctly.

**Comment by bablokb** [comment]
It is an UC8151-chip. I will create a PR which fixes the board.c by setting grayscale to true.

**Comment by tannewt** [comment]
I think #7777 may fix this more broadly. (I saw [a similar issue on the forum](https://forums.adafruit.com/viewtopic.php?t=199888).)

**Comment by bablokb** [comment]
Fixed by #7777
